java/src/main/java/com/scalar/admin/AdminClient.java

```diff
@@ -22,6 +27,30 @@ public AdminClient(String host, int port) {
     this.blockingStub = AdminGrpc.newBlockingStub(channel);
   }
 
+  public AdminClient(
```

Comment:
this logic is a copycat of https://github.com/scalar-labs/scalardb-cluster/blob/v3.12.0/common/src/main/java/com/scalar/db/cluster/common/RemoteClusterNode.java#L55